src/index.js

```diff
@@ -83,6 +109,14 @@ const date = new Date().toISOString().match(/\d+-\d+-\d+/)[0];
 
 const changelog = rawChangelog.map(l => l.replace(/^\+/, ''));
 
+const codeowners = getCodeowners();
+let ccList = ['cc:'];
```

Comment:
How about having a separate `codeowners: ...` line (above the cc)?
This would make it clear why each user was added and whether the user requesting the deploy actually wanted to add any extra explicit cc.